packages/ilios-common/addon/components/html-editor.js

```diff
@@ -28,12 +28,25 @@ export default class HtmlEditorComponent extends Component {
     this.editorId = guidFor(this);
   }
 
+  @cached
+  get createEditor() {
```

Comment:
I'm having second thoughts about this name. Should probably be `loadFroala` `createdEditor` is a little difficult to understand when reading, especially the template.